src/main/resources/config/liquibase/changelog/20200726015816_changelog.xml

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.1" encoding="UTF-8" standalone="no"?>
 <databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-3.9.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.9.xsd">
-    <changeSet author="birtan (generated)" id="1595721518853-31">
-        <createTable tableName="text_assessment_conflict">
+    <changeSet author="birtan" id="20200726015816">
+        <createTable tableName="feedback_conflict">
             <column autoIncrement="true" name="id" type="BIGINT">
-                <constraints nullable="false" primaryKey="true" primaryKeyName="text_assessment_conflictPK"/>
+                <constraints nullable="false" primaryKey="true" primaryKeyName="assessment_conflictPK"/>
```

Comment:
Why is the name here not something like `feedback_conflictPK`?

Reply:
good point, we should adapt it

Reply:
done